### PR TITLE
[SWTableViewCell] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: iOS/SWTableViewCell/component/component.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 id: swtableviewcell
-version: 0.3.7.0
+version: 0.3.7
```
Comment (bholmes) [inline iOS/SWTableViewCell/component/component.yaml]: Bump this to 0.3.7.1

File: iOS/SWTableViewCell/build.cake [hunk 2]
```diff
@@ -4,7 +4,7 @@
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
 var IOS_PODS = new List<string> {
-	"platform :ios, '6.0'",
+	"platform :ios, '11.2'",
```
Comment (SotoiGhost) [inline iOS/SWTableViewCell/build.cake]: Does this pod support only the most recent iOS version?

File: iOS/SWTableViewCell/nuget/Xamarin.SWTableViewCell.nuspec [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <metadata>
+    <id>Xamarin.SWTableViewCell</id>
+    <title>SWTableViewCell for Xamarin.iOS</title>
+    <version>0.3.7</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>An easy-to-use UITableViewCell subclass that implements a swipeable content view which exposes utility buttons (similar to iOS 7 Mail Application).</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://components.xamarin.com/view/swtableviewcell</projectUrl>
```
Comment (newky2k) [inline iOS/SWTableViewCell/nuget/Xamarin.SWTableViewCell.nuspec]: Needs to be an FWLink

File: iOS/SWTableViewCell/nuget/Xamarin.SWTableViewCell.nuspec [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <licenseUrl>https://components.xamarin.com/license/swtableviewcell</licenseUrl>
```
Comment (newky2k) [inline iOS/SWTableViewCell/nuget/Xamarin.SWTableViewCell.nuspec]: Needs to be an FWLink

**Review by newky2k — CHANGES_REQUESTED** [review]
You should also specify nobuild: true on the sample and no_build to stoe the source being built in the component.yaml

Also does this need a external dependencies file for the cocoapod

File: iOS/SWTableViewCell/component/component.yaml [hunk 5]
```diff
@@ -12,6 +12,10 @@ libraries:
 samples:
   - name: "iOS Unified Sample"
     path: ../samples/SWTableViewCellSample.sln
+    installNuGets:
```
Comment (newky2k) [inline iOS/SWTableViewCell/component/component.yaml]: Missing remove part of the nuget installation.  should remove the source project

File: iOS/SWTableViewCell/component/component.yaml [hunk 6]
```diff
@@ -12,6 +12,10 @@ libraries:
 samples:
   - name: "iOS Unified Sample"
     path: ../samples/SWTableViewCellSample.sln
+    installNuGets:
+      - project: SWTableViewCellSample
+        packages:
+          - Xamarin.SWTableViewCell
     configuration: "Release|iPhone"
```
Comment (newky2k) [inline iOS/SWTableViewCell/component/component.yaml:23]: need to add the local-nuget-repo: ../output and is_shell: true

**Review by newky2k — CHANGES_REQUESTED** [review]

File: iOS/SWTableViewCell/component/component.yaml (hunk 6 above)
Comment (newky2k) [reply iOS/SWTableViewCell/component/component.yaml:23]: no externals file in the component.yaml

**Comment by newky2k** [comment]
It fails to build the component becuase it cannot install the nuget into the sample, becuase the local-nuget-repo is not provided

File: iOS/SWTableViewCell/component/component.yaml [hunk 7]
```diff
@@ -1,17 +1,24 @@
...
 summary: An easy-to-use UITableViewCell subclass that implements a swipeable content view which exposes utility buttons (similar to iOS 7 Mail Application).
+license: ../License.md
 
 libraries:
   ios-unified: ../output/unified/SWTableViewCell.dll
 
 samples:
   - name: "iOS Unified Sample"
-    path: ../samples/SWTableViewCellSample.sln
+    path: ../samples/SWTableViewCellSample.
```
Comment (newky2k) [inline iOS/SWTableViewCell/component/component.yaml]: missing the file extensions on the sample solution

**Review by newky2k — CHANGES_REQUESTED** [review]

File: iOS/SWTableViewCell/component/component.yaml [hunk 8]
```diff
@@ -1,17 +1,24 @@
 id: swtableviewcell
-version: 0.3.7.0
+version: 0.3.7.1
 name: SWTableViewCell
 publisher: Xamarin Inc
 publisher-url: http://xamarin.com/
 src-url: https://github.com/xamarin/XamarinComponents/tree/master/iOS/SWTableViewCell
 summary: An easy-to-use UITableViewCell subclass that implements a swipeable content view which exposes utility buttons (similar to iOS 7 Mail Application).
+license: ../License.md
```
Comment (newky2k) [inline iOS/SWTableViewCell/component/component.yaml:8]: License.md is not in the root folder so the component will not compile

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — APPROVED** [review]

File: iOS/SWTableViewCell/samples/SWTableViewCellSample/.vs/SWTableViewCellSample/xs/UserPrefs.xml [hunk 9]
```diff
@@ -0,0 +1,20 @@
+﻿<Properties StartupConfiguration="{424FB004-7068-44D5-880E-CB76B7B8A653}|Default">
```
Comment (newky2k) [inline iOS/SWTableViewCell/samples/SWTableViewCellSample/.vs/SWTableViewCellSample/xs/UserPrefs.xml]: This file needs to be removed, as well as the .vs folder in general

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — APPROVED** [review]

**Pull request merged**